src/TwigFilter.php

```diff
@@ -29,7 +29,7 @@ final class TwigFilter
     private $arguments = [];
 
     /**
-     * @param callable|array<class-string, string>|null $callable A callable implementing the filter. If null, you need to overwrite the "node_class" option to customize compilation.
+     * @param callable|array{class-string, string}|null $callable A callable implementing the filter. If null, you need to overwrite the "node_class" option to customize compilation.
```

Comment:
Is `array{class-string, string}` as a shortcut for `array{0: class-string, 1: string}` understood by all import static analyzers (Psalm, PhpStan, PhpStorm)?

Reply:
AFAIK yes

Reply:
AFAIK too, yes.

But if you prefer (to be explicit ?) I can use `array{0: class-string, 1: string}`